### ISSUE TITLE
KvsAll and AllvsAll label smoothing can produce targets above 1

### ISSUE THREAD

**Issue by LckyLke** [dice-group/dice-embeddings#433]
With two entities and smoothing 0.1, labels should be [0.05, 0.95]. Instead, they are [0.5, 1.4]. The uniform term is missing the smoothing multiplier. Both datasets are affected.

Reproduced on `66a7e619`.

```python
import numpy as np
import torch
from dicee.dataset_classes import KvsAll, AllvsAll

for cls in (KvsAll, AllvsAll):
    kwargs = {'form': 'EntityPrediction'} if cls is KvsAll else {}
    ds = cls(np.array([[0, 0, 1]]), {'a': 0, 'b': 1}, {'r': 0},
             label_smoothing_rate=0.1, **kwargs)
    labels = ds[0][1]
    print(cls.__name__, labels.tolist(), '(expected [0.05, 0.95])')
    torch.testing.assert_close(labels, torch.tensor([0.05, 0.95]))
```
